client/src/components/CardContainer/index.js

```diff
@@ -0,0 +1,3 @@
+import CardContainer from "./CardContainer";
```

Comment:
Please, remove this file and transfer your re-export to /components/index.js file.

Reply:
done